__tests__/__integration__/hello_world/tsconfig.json

```diff
@@ -1,38 +1,37 @@
 {
-    "compilerOptions": {
-      "types": [
-        "node",
-        "jest"
-      ],
-      "lib": [
-        "esnext"
-      ],
-      "target": "es2015",
-      "module": "commonjs",
-      "declaration": true,
-      "moduleResolution": "node",
-      "noImplicitAny": true,
-      "outDir": "./lib",
-      "preserveConstEnums": true,
-      "removeComments": false,
-      "pretty": true,
-      "sourceMap": true,
-      "newLine": "lf"
-    },
-    "typeRoots": [
-      "./node_modules/@types"
+  "compilerOptions": {
+    "types": [
+      "node",
+      "jest"
     ],
-    "include": [
-      "./src",
-      "../../../packages/**/*"
+    "lib": [
+      "esnext"
     ],
-    "exclude": [
-      "lib",
-      "node_modules",
-      "../../../packages/**/__mocks__/*",
-      "../../../packages/**/__tests__/*",
-      "../../../packages/**/*.spec.ts",
-      "../../../packages/**/*-spec.ts",
-      "../../../packages/**/*.test.ts"
-    ]
-  }
\ No newline at end of file
+    "target": "es2015",
+    "module": "commonjs",
+    "declaration": true,
+    "moduleResolution": "node",
+    "noImplicitAny": true,
+    "outDir": "./lib",
+    "preserveConstEnums": true,
+    "removeComments": false,
+    "pretty": true,
+    "sourceMap": true,
+    "newLine": "lf"
+  },
+  "typeRoots": [
+    "./node_modules/@types"
+  ],
+  "include": [
+    "./src"
+  ],
+  "exclude": [
+    "lib",
+    "node_modules",
+    "../../../packages/**/__mocks__/*",
+    "../../../packages/**/__tests__/*",
+    "../../../packages/**/*.spec.ts",
+    "../../../packages/**/*-spec.ts",
+    "../../../packages/**/*.test.ts"
+  ]
```

Comment:
Probably a dumb question, ...
doesn't `hello_world` need the `references` as well?

Reply:
Not a dumb question, I was dumb here 😁